CHANGELOG.md

```diff
@@ -1,26 +1,268 @@
+
+These pages are still under development as of this release, but you can view the current progress at:
+
+https://develop.pwa-venia.com/cart
+
+### Order Confirmation page
+
+This release adds an Order Confirmation page at the end of the cart and checkout workflow.
+It contains the billing and shipping information specified during checkout along with the items ordered.
+
+This page works with guest and authenticated checkout.
+For guest checkout, the shopper is given the option to create an account for the store.
+
+### Standard Dialog component
+
+This release introduces a standard modal window with the [Dialog component][].
+
+Modals are child windows that render over the main application.
+They are highly visual components that show important messages or prompts for user interactions.
+
+The Dialog component introduced in this release provides a standard way of working with this modal windows.
+This guarantees a unified look and feel for all your modal window use cases.
+
+### PWA Studio Fundamentals tutorials
+
+Over the course of 7.0.0 development, the PWA Studio doc site has published introductory tutorials for working with the PWA-Studio tools and libraries.
+
+These tutorials provide steps for common tasks associated with storefront development.
+They cover everything from setting up the initial project to providing a checklist for deploying to production.
+
+See [PWA Studio fundamentals][] for a list of these tutorials.
+
+## Pull requests merged in this release
+
+### Venia (storefront and visual component library)
+
+| Description                                                                             | Change type | PR        |
+| --------------------------------------------------------------------------------------- | ----------- | --------- |
+| Created a new Portal component                                                          | **Feature** | [#2436][] |
+| Added support for png image requests for servers that cannot handle `webp`              | **Feature** | [#2400][] |
+| Implemented UX around $0 total checkout                                                 | **Feature** | [#2394][] |
+| Implemented auto-population of shipping information when authenticated                  | **Feature** | [#2380][] |
+| Implemented shopping cart merging on login                                              | **Feature** | [#2377][] |
+| Added ability to specify ratio for image component                                      | **Feature** | [#2372][] |
+| Implemented Checkout Page price adjustments                                             | **Feature** | [#2366][] |
+| Created a new Dialog Component                                                          | **Feature** | [#2365][] |
+| Added storybook to venia concept for scaffolded projects                                | **Feature** | [#2355][] |
```

Comment:
"Venia" should be capitalized and I *think* that "Storybook" should be capitalized as well.